config.json

```diff
@@ -517,6 +523,12 @@
             "ContainerName": "default"
           }
         },
+        "nodeReuse":{
```

Comment:
It can't be the default as it depends on what OS we are running run.exe from and the defaults values are always passed. There is not a validation that according to the OS use one setting or another.